airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/unstructured_parser.py

```diff
@@ -54,6 +54,14 @@ def parser_max_n_files_for_parsability(self) -> Optional[int]:
         """
         return 0
 
+    def get_parser_defined_primary_key(self, config: FileBasedStreamConfig) -> Optional[str]:
+        """
+        Return the document_key field as the primary key.
+
+        his will pre-select the document key column as the primary key when setting up a connection, making it easier for the user to configure normalization in the destiantion.
```

Comment:
```suggestion
        This will pre-select the document key column as the primary key when setting up a connection, making it easier for the user to configure normalization in the destination.
```